### ISSUE TITLE
Can't open the app at all.

### ISSUE THREAD

**Issue by The-Pan-Man** [alievk/avatarify-desktop#15]
I downloaded and ran the installer, The app appears on my desktop, but every time I try to run it I get this message. I have re-installed it multiple times. 
![image](https://user-images.githubusercontent.com/43502300/117065783-e5f5fa80-acf5-11eb-8e25-e3369f11d4a2.png)

**Comment by staddy** [comment]
@The-Pan-Man, you probably have 32 bit system, unfortunately Avatarify Desktop doesn't support it

**Comment by The-Pan-Man** [comment]
My computer says its 64 bit. 
![image](https://user-images.githubusercontent.com/43502300/117667322-c8e87e00-b172-11eb-8dce-c4c47a945fc7.png)
I'm using a Surface Go if that means anything.

**Comment by JohanAR** [comment]
> My computer says its 64 bit.
> ![image](https://user-images.githubusercontent.com/43502300/117667322-c8e87e00-b172-11eb-8dce-c4c47a945fc7.png)
> I'm using a Surface Go if that means anything.

Yes, you need an Nvidia GPU to run Avatarify, preferably a GTX 1070 or better, it's in the requirements.
